### ISSUE TITLE
[Accessibility] MAS 4.2.1: (eclipse/tool bar '') The Name property of a focusable element must not be null.

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#4290]
The following accessibility issue needs investigation.

[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-Benchmark; #MAS4.2.1; #JavatoolingforIntelliJ/Eclipse; #C+E-H; #Benchmark; #A11ySev2; #A11yAuto; #AI4D 

**NOTE:** Same issue is observed in all other tabs also

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2020-03 (4.15.0)
Build id: 20200313-1211

**Solution Installed:** The Azure Toolkit for Eclipse 3.220, Scala IDE 4.7.x

**OS Details:**
OS: Windows (Version 1903, Build: 18362.175)

**Screen Reader:** Narrator , NVDA

**MAS Violated:** MAS 4.2.1

**App:** eclipse

**Element path:** tool bar ''

**Issue Details:** The Name property of a focusable element must not be null. [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology)

**How To Fix:** Provide a UI Automation Name property that concisely identifies the element.

This accessibility issue was found using Accessibility Insights for Windows, a tool that helps debug and find accessibility issues earlier. Get more information and download this tool at https://aka.ms/AccessibilityInsights.

**MAS Reference:**
[MAS 4.2.1](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7BEEE97BBD-9990-48A0-8B03-6EA775581AA8%7D&file=MAS%204.2.1%20%E2%80%93%20Object%20Information.docx&action=default&mobileredirect=true)

![Bug_4290_Java Tooling_Module Dependencies tab_AI4D](https://user-images.githubusercontent.com/41108985/79010330-5f68fb00-7b7f-11ea-96ac-cbe3d79aedc4.png)
![Bug_4290_Java Tooling_Order and Export tab_AI4D](https://user-images.githubusercontent.com/41108985/79010340-6263eb80-7b7f-11ea-84e6-a1d549c87d62.png)
![Bug_4290_Java Tooling_Libraries tab_AI4D](https://user-images.githubusercontent.com/41108985/79010343-64c64580-7b7f-11ea-9eb9-e388e817f15b.png)
![Bug_4290_Java Tooling_Projects tab_AI4D](https://user-images.githubusercontent.com/41108985/79010346-66900900-7b7f-11ea-8193-fbf395a019ce.png)
![Bug_4290_Java Tooling_Source tab_AI4D](https://user-images.githubusercontent.com/41108985/79010352-68f26300-7b7f-11ea-8a12-4a19b987498a.png)

**Comment by jingyanjingyan** [comment]
This wizard is extends from original Scala Plugin. You can find the same dialog in New Project > Scala Wizards > Scala Project. So it's out of the scope of our toolkit. Please find the screenshot of Eclipse\ScalaPlugin without our toolkit.

One of the tabs:
![image](https://user-images.githubusercontent.com/36756024/81462307-33738080-91e4-11ea-9317-2980564c8b2f.png)

**Comment by jingyanjingyan** [comment]
+ Importance of experience: Important. When the Name property of a focusable element is null, narrator could not read what's the element used for. 
+ Quality of experience: Annoying. User could tab to the certain element that used to create project, and narrator could read them. It will not block create process.
+ Alternate solutions: This issue will not break users experience, no need for alternate solutions
+ Evidence: All evidence is provided in this issue.
+ [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

External issues tracked in: https://bugs.eclipse.org/bugs/show_bug.cgi?id=562986

**Comment by Alexandria-R** [comment]
Known eclipse issue, approved to close

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749646
